### PR TITLE
Digest #20

### PR DESCRIPTION
**Pull request by pjazdzewski1990**

### CODE REVIEW

File: _posts/25-02-2015-digest20.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+tags:
+- Scalac Weekly Digest
+---
+
+Hi folks!
+
+
+
+<p id="1"></p>
+* \[Scala\] [Comparing Elasticity of Reactive Frameworks](http://www.infoq.com/presentations/elasticity-reactive-frameworks)
```
Comment (andrjew) [inline _posts/25-02-2015-digest20.md]: :+1:

Comment (tomaszperek) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+Hi folks!
+
+
+
+<p id="1"></p>
+* \[Scala\] [Comparing Elasticity of Reactive Frameworks](http://www.infoq.com/presentations/elasticity-reactive-frameworks) 
+* \[Gaming\] [Bunnyhopping from the Programmer's Perspective](https://flafla2.github.io/2015/02/14/bunnyhop.html)
+* \[Javascript\] [The Birth & Death of JavaScript](https://www.destroyallsoftware.com/talks/the-birth-and-death-of-javascript)
```
Comment (tomaszperek) [inline _posts/25-02-2015-digest20.md]: :+1:  funny;)

File: _posts/25-02-2015-digest20.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Scalac Weekly Digest
+---
+
+Hi folks!
+
+
+
+<p id="1"></p>
+* \[Scala\] [Comparing Elasticity of Reactive Frameworks](http://www.infoq.com/presentations/elasticity-reactive-frameworks) 
+* \[Gaming\] [Bunnyhopping from the Programmer's Perspective](https://flafla2.github.io/2015/02/14/bunnyhop.html)
```
Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: :-1: solution for particular problem in particular kind of games.

File: _posts/25-02-2015-digest20.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+* \[Javascript\] [The Birth & Death of JavaScript](https://www.destroyallsoftware.com/talks/the-birth-and-death-of-javascript)
+* \[Javascript\] [Testing React Web Apps with Mocha](http://www.hammerlab.org/2015/02/14/testing-react-web-apps-with-mocha/)
```
Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: :+1: interesting and informative

File: _posts/25-02-2015-digest20.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+* \[Javascript\] [60 FPS on the mobile web](http://engineering.flipboard.com/2015/02/mobile-web/)
```
Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: :+1: set of interesting tips

File: _posts/25-02-2015-digest20.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+* \[Technology\] [When expotantial progress becomes reality] https://t.co/fQBpir1POC
+* \[Brain/Health\] [New answer to a question why do we sleep?] http://www.ted.com/talks/jeff_iliff_one_more_reason_to_get_a_good_night_s_sleep
+* \[Clojure\] [Stuart Halloway: Introduction to Clojure](https://vimeo.com/68375202)
```
Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: :+1: every clojure learning source is well seen :) imo quite better that paid udemy course.

File: _posts/25-02-2015-digest20.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+* \[Architecture\] [Thinking Fast and Slow with Software Development](http://www.slideshare.net/dbryant_uk/thinking-fast-and-slow-2015-jfokus-final)
+* \[Scala\] [Composing a domain workflow with statically checked invariants](http://debasishg.blogspot.com/2015/02/functional-patterns-in-domain-modeling.html)
```
Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: :+1: :-1: It's good, but only for simple and linear workflows. From other side there is no tool for workflow in scala.

File: _posts/25-02-2015-digest20.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+* \[Scala\] [Scaling With Akka Streams: Fetch 1M Twitter Profiles in Less Than 5min](http://blog.abhinav.ca/blog/2015/02/19/scaling-with-akka-streams/)
+* \[Functional Programming\] [The World Is a Functional Program](http://www.baweaver.com/blog/2015/02/20/the-world-is-a-functional-program/)
```
Comment (LukasGasior1) [inline _posts/25-02-2015-digest20.md]: :+1:

Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 3 above)
Comment (pjazdzewski1990) [inline _posts/25-02-2015-digest20.md]: :-1:

File: _posts/25-02-2015-digest20.md (hunk 4 above)
Comment (pjazdzewski1990) [inline _posts/25-02-2015-digest20.md]: well explained :+1:

File: _posts/25-02-2015-digest20.md (hunk 5 above)
Comment (pjazdzewski1990) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
+<p id="1"></p>
+* \[Scala\] [Comparing Elasticity of Reactive Frameworks](http://www.infoq.com/presentations/elasticity-reactive-frameworks) 
+* \[Gaming\] [Bunnyhopping from the Programmer's Perspective](https://flafla2.github.io/2015/02/14/bunnyhop.html)
+* \[Javascript\] [The Birth & Death of JavaScript](https://www.destroyallsoftware.com/talks/the-birth-and-death-of-javascript)
+* \[Javascript\] [Testing React Web Apps with Mocha](http://www.hammerlab.org/2015/02/14/testing-react-web-apps-with-mocha/)
+* \[Javascript\] [60 FPS on the mobile web](http://engineering.flipboard.com/2015/02/mobile-web/)
+* \[Technology\] [When expotantial progress becomes reality] https://t.co/fQBpir1POC
```
Comment (pjazdzewski1990) [inline _posts/25-02-2015-digest20.md]: Awesome  :+1:

Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 2 above)
Comment (szabba) [inline _posts/25-02-2015-digest20.md]: :+1: Talki z Destory all Software generalnie są fajne.

Comment (szabba) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 1 above)
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: :-1:

File: _posts/25-02-2015-digest20.md (hunk 4 above)
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 5 above)
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 9 above)
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 5 above)
Comment (szabba) [inline _posts/25-02-2015-digest20.md]: :+1: For not ignoring accesibility.

File: _posts/25-02-2015-digest20.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+* \[Brain/Health\] [New answer to a question why do we sleep?] http://www.ted.com/talks/jeff_iliff_one_more_reason_to_get_a_good_night_s_sleep
```
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: :-1: when it comes to scientific theories I'd rather see a blog post or scientific paper with relevant links to the source

File: _posts/25-02-2015-digest20.md (hunk 1 above)
Comment (pslaski) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 4 above)
Comment (pslaski) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 8 above)
Comment (pslaski) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+* \[Clojure\] [Stuart Halloway: Introduction to Clojure](https://vimeo.com/68375202)
+* \[Architecture\] [Thinking Fast and Slow with Software Development](http://www.slideshare.net/dbryant_uk/thinking-fast-and-slow-2015-jfokus-final)
```
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: :-1:

File: _posts/25-02-2015-digest20.md (hunk 7 above)
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: :+1: it's about functional patterns not workflow tools

File: _posts/25-02-2015-digest20.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+* \[Scala\] [Composing a domain workflow with statically checked invariants](http://debasishg.blogspot.com/2015/02/functional-patterns-in-domain-modeling.html)
+* \[Scala\] [Scaling With Akka Streams: Fetch 1M Twitter Profiles in Less Than 5min](http://blog.abhinav.ca/blog/2015/02/19/scaling-with-akka-streams/)
```
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: :+1:

File: _posts/25-02-2015-digest20.md (hunk 7 above)
Comment (pjazdzewski1990) [inline _posts/25-02-2015-digest20.md]: IMO a fancy name for something quite simple, still worth mentioning :+1:

File: _posts/25-02-2015-digest20.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+title: Scalac Weekly Digest#20
+summary: Collection of interesting links from around of web chosen for you by Scalac team
+author: Scalac Team
+tags:
+- Scalac Weekly Digest
+---
+
+Hi folks!
+
+Welcome to 20th edition of Scalac Weekly digest. We are really happy that you are reading us for over 5 months now. This week we focused on providinf you with programming related material. We selected for you the most interesting links about Scala, Javascript and Clojure, so everyone will find something interesting for them. See you next week
```
Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: providin**g** , material[s]?

File: _posts/25-02-2015-digest20.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+
+<p id="1"></p>
+* \[Scala\] [Comparing Elasticity of Reactive Frameworks](http://www.infoq.com/presentations/elasticity-reactive-frameworks) 
+
+In his talk James Ward takes a look at Reactive framework and tries to compare them against each other. Interesting, especially for those lost in the marketing noise.
```
Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: lost -> who lost ?

File: _posts/25-02-2015-digest20.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+
+This article is a funny and insightful look on functional programming and the traits it was built upon.
+
+<p id="8"></p>
+* \[Clojure\] [Stuart Halloway: Introduction to Clojure](https://vimeo.com/68375202)
+
+A "must watch" video for everyone interested in Clojure and Functional Programming as a whole. 
+
+<p id="9"></p>
+* \[Technology\] [When expotantial progress becomes reality](https://t.co/fQBpir1POC)
```
Comment (marioosh) [inline _posts/25-02-2015-digest20.md]: expotantial? :) -> Exponential

File: _posts/25-02-2015-digest20.md (hunk 13 above)
Comment (serafinmaciej) [inline _posts/25-02-2015-digest20.md]: We selected the most interesting links about Scala, Javascript and Clojure *for you*,...

File: _posts/25-02-2015-digest20.md (hunk 14 above)
Comment (serafinmaciej) [inline _posts/25-02-2015-digest20.md]: zagubionych ;)

File: _posts/25-02-2015-digest20.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+
+<p id="2"></p>
+* \[Scala\] [Composing a domain workflow with statically checked invariants](http://debasishg.blogspot.com/2015/02/functional-patterns-in-domain-modeling.html) 
+
+This blogpost covers creating a domain workflow using Kleisli implemented in scalaz library. Instructive for anyone interested in functional patterns and providing maximum type safety.
+
+<p id="3"></p>
+* \[Scala\] [Scaling With Akka Streams: Fetch 1M Twitter Profiles in Less Than 5min](http://blog.abhinav.ca/blog/2015/02/19/scaling-with-akka-streams/)
+
+Akka Streams project was created as a way of handling large ammount of data. The link above is an example on how to use this technology for tasks like that, by doing the "Big Data Hello World" application - indexing Tweets.
```
Comment (serafinmaciej) [inline _posts/25-02-2015-digest20.md]: for such tasks

File: _posts/25-02-2015-digest20.md (hunk 13 above)
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: **the** 20th edition of **the** Scalac Weekly digest

File: _posts/25-02-2015-digest20.md [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+
+<p id="4"></p>
+* \[Javascript\] [The Birth & Death of JavaScript](https://www.destroyallsoftware.com/talks/the-birth-and-death-of-javascript)
+
+Wait, what? Death of JavaScript? Well, not exactly. Funny yet interesting talk by Gary Bernhardt about the history of JavaScript from 1995 until 2035.
+
+<p id="5"></p>
+* \[Javascript\] [Testing React Web Apps with Mocha](http://www.hammerlab.org/2015/02/14/testing-react-web-apps-with-mocha/) 
+
+Most annoying thing in developers world is to wait for something... If you want to tests ReactJs fastly (maybe even do some TDD) with tools you know, this article is for you.
```
Comment (serafinmaciej) [inline _posts/25-02-2015-digest20.md]: fast...? fastly jest zdaje sie obsolete

File: _posts/25-02-2015-digest20.md [hunk 18]
```diff
@@ -0,0 +1,60 @@
...
+
+<p id="6"></p>
+* \[Javascript\] [60 FPS on the mobile web](http://engineering.flipboard.com/2015/02/mobile-web/) 
+
+Building cross platform applications was always a hot topic (write once, run anywhere). The bigest problem is a hybrid app performance... The solution provided by flipboard developers is just awesome! They are using Canvas with ReactJs to deliver Native-like experience - must-read position!
```
Comment (serafinmaciej) [inline _posts/25-02-2015-digest20.md]: has always been a hot topic

File: _posts/25-02-2015-digest20.md (hunk 13 above)
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: We are really happy that you are reading us for over 5 months now -> We are really happy that you **have been with us for the last five months**

Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: This week we **are** focusing on providin**g** you with **a** programming related material ->

Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: We selected for you -> We**'ve** selected for you

File: _posts/25-02-2015-digest20.md (hunk 14 above)
Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: Reactive framework -> Reactive framework**s**

Comment (jczuchnowski) [inline _posts/25-02-2015-digest20.md]: Interesting, especially -> usunąć przecinek albo wstawić kropkę

**Pull request merged**